build.gradle

```diff
@@ -7,7 +7,7 @@ def properties(String key) {
 plugins {
     id "java"
     id "org.jetbrains.kotlin.jvm" version "1.6.10"
-    id "org.jetbrains.intellij" version "1.4.0"
+    id "org.jetbrains.intellij" version "1.6.0"
```

Comment:
Needed to upgrade the Intellij plugin because the `runIde` gradle step dies if not upgraded to this version. Hurray backwards compatibility 🎉